cmd/logger/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2"
+	"net/http"
+	"net/url"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/kubeflow/kfserving/pkg/logger"
+	"github.com/pkg/errors"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var (
+	logUrl    = flag.String("log_url", "", "The URL to send request/response logs to")
+	port      = flag.String("port", "8080", "Executor port")
+	svcPort   = flag.String("svc_port", "8081", "The local port of the service")
+	workers   = flag.Int("workers", 5, "Number of workers")
+	sourceUri = flag.String("source_uri", "", "The source URI to use when publishing cloudevents")
+	logMode   = flag.String("log_mode", "all", "Whether to log 'request', 'response' or 'all'")
+	modelId   = flag.String("model_id", "", "The model ID to add as header to log events")
+)
+
+func main() {
+	flag.Parse()
+
+	logf.SetLogger(logf.ZapLogger(false))
+	log := logf.Log.WithName("entrypoint")
+
+	if *logUrl == "" {
+		log.Info("logUrl argument must not be empty.")
+		os.Exit(-1)
+	}
+
+	logUrlParsed, err := url.Parse(*logUrl)
+	if err != nil {
+		log.Info("Malformed log_url", "URL", *logUrl)
+		os.Exit(-1)
+	}
+	var loggingType v1alpha2.LoggerMode
+	if *logMode == string(v1alpha2.LogAll) {
```

Comment:
I'd need to play with it, but I think the idiomatic way to do this in go is like this https://yourbasic.org/golang/iota/

Reply:
> Nice work! Making my way through the PR in pieces.
> 
> Looks like we might be missing cloud builders for the logger.

I addded
`release/triggers/latest/logger.yaml`
`release/triggers/tagged/logger.yaml`

Reply:
Updates to a switch statement